### ISSUE TITLE
Add VCR-like functionality

### ISSUE THREAD

**Issue by petRUShka** [Huddle/PhantomXHR#13]
There is great Ruby tool VCR(https://github.com/vcr/vcr) that provide caching requests instead of mocking.

> Record your test suite's HTTP interactions and replay them during future test runs for fast, deterministic, accurate tests.

It would be great to have such functionality in PhantomXHR.

**Comment by jamescryer** [comment]
I agree, this would be a great feature.  Record and 'export' of both XHR and user events would be incredibly useful.  Would really love to see a PR for this

**Comment by sulabhjain** [comment]
+1
